kolibri/utils/options.py

```diff
@@ -71,6 +71,11 @@
             "default": "https://studio.learningequality.org",
             "envvars": ("KOLIBRI_CENTRAL_CONTENT_BASE_URL", "CENTRAL_CONTENT_DOWNLOAD_BASE_URL",),
         },
+        "SUBPATH_MOUNT_URL": {
+            "type": "string",
+            "default": "",
+            "envvars": ("KOLIBRI_SUBPATH_MOUNT_URL", ),
```

Comment:
Talked it through with @jamalex and I think we managed to come up with something that shouldn't need to mix with the Django settings - if someone decides to set the STATIC_URL to a URI outside of Kolibri, then that will take precedence.

For this PATH_PREFIX it will only ever be a prefix within the current server, and so will not be a URI.

Reply:
Will also add to STATIC_URL in base settings but yeah we will let that be overridden. Caveat usor.